nextflow_schema.json

```diff
@@ -103,6 +103,10 @@
                     "description": "Naming of sequencing files",
                     "help_text": "Indicates the naming of sequencing files (default: `\"/*_R{1,2}_001.fastq.gz\"`).\n\nPlease note:\n\n1. The prepended slash (`/`) is required\n2. The star (`*`) is the required wildcard for sample names\n3. The curly brackets (`{}`) enclose the orientation for paired end reads, seperated by a comma (`,`).\n4. The pattern must be enclosed in quotes\n\nFor example for one sample (name: `1`) with forward (file: `1_a.fastq.gz`) and reverse (file: `1_b.fastq.gz`) reads in folder `data`:\n\n```bash\n--input \"data\" --extension \"/*_{a,b}.fastq.gz\"\n```"
                 },
+                "picrust": {
+                    "type": "boolean",
+                    "description": "If functional abundances are predicted from the marker genes."
```

Comment:
"Functional gene abundances predicted from phylogenetic relationships to marker gene organisms."?

Reply:
Changed.